utils/cliutils/commandsflags.go

```diff
@@ -1554,7 +1554,7 @@ var flagsMap = map[string]cli.Flag{
 	},
 	repository: cli.StringFlag{
 		Name:  repository,
-		Usage: "[Optional] Repository name to filter resource.` `",
+		Usage: "Repository name to filter resource.` `",
```

Comment:
Repository name to filter resource
-->
[Mandatory] Repository name to filter the resource by.